recipes/compyle/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "compyle" %}
+{% set version = "0.6" %}
```

It appears `0.6` version doesn't exist upstream?
```suggestion
{% set version = "0.5" %}
```
Version numbers need to match with upstream else it will make for a confusing mess.

---

The master branch was way ahead of the v0.5 tag, so I thought of stating it as 0.6 instead. i have corrected it back to v0.5 in the new recipe(force-pushed)